anitya/templates/project_new.html

```diff
@@ -253,6 +262,7 @@ <h1>{{ context }} project</h1>
   }
 
   $(document).ready(function() {
+    add_tooltips();
```

Comment:
This change seems unrelated to sorting by creation date - should it be a separate pull request?

Reply:
I did this only for small optimalization, but you are right, I will remove it from this PR.